rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.72"
+channel = "1.74"
```

Comment:
Needed to fix cargo audit: https://app.circleci.com/pipelines/github/influxdata/pbjson/274/workflows/b620d772-632d-4dd8-bc83-cd9a0e01a30b/jobs/1433